examples/reference_environments/airflow_2_6_2/compose.yaml

```diff
@@ -167,6 +169,17 @@ services:
       airflow-init:
         condition: service_completed_successfully
 
+  data_docs:
+    # airflow-common is added so that the volumes config is available to the data_docs container
+    <<: *airflow-common
```

Comment:
This was new to me, these are YAML anchors: https://stackoverflow.com/a/45805673.